### PR TITLE
chore: Ported readme file to new readme format

### PR DESCRIPTION
**Pull request by mark1000**
Current draft is mostly Mike Pope's draft with a couple tweaks:
- removed hyperlink to self, per his comment
- kept exported markdown tables

### CODE REVIEW

File: 0-bootstrap/README.md [hunk 1]
```diff
@@ -1,59 +1,146 @@
...
-  -  If this is the case, try the following:
-      ```
-      gcloud auth application-default login
-      gcloud auth list # <- confirm that correct account has a star next to it
-      ```
-  - Re-run `terraform` after.
-- `Error: Error setting billing account "XXXXXX-XXXXXX-XXXXXX" for project "projects/some-project": googleapi: Error 400: Precondition check failed., failedPrecondition`. Most likely this is related to billing quota issue.
-  - To confirm this, try `gcloud alpha billing projects link projects/some-project --billing-account XXXXXX-XXXXXX-XXXXXX`.
-  - If output states `Cloud billing quota exceeded`, please request increase via [https://support.google.com/code/contact/billing_quota_increase](https://support.google.com/code/contact/billing_quota_increase).
-
```
Comment (bharathkkb) [inline 0-bootstrap/README.md:30]: Lets make sure to preserve all this troubleshooting info in the bottom section until we have a separate doc

**Review by bharathkkb — COMMENTED** [review]
Thanks for the PR @mark1000 

/cc @rjerrems @daniel-cit for another pass

File: 0-bootstrap/README.md [hunk 2]
```diff
@@ -1,59 +1,146 @@
...
+</tr>
+</tbody>
+</table>
+
+For an overview of the architecture and the parts, see the
+`[terraform-example-foundation` README file](https://github.com/terraform-google-modules/terraform-example-foundation).
+
+## Purpose
+
+The purpose of this step is to [...].
```
[the remark this answers is not in the document]
Comment (mark1000) [reply 0-bootstrap/README.md:55]: This is from Mike's draft. I don't think he knew what to put here, and I didn't either. 
What will be produced if someone runs the Terraform in step 0 here?

File: 0-bootstrap/README.md [hunk 3]
```diff
@@ -1,59 +1,146 @@
...
-
-If you are using the `jenkins_bootstrap` sub-module, please see [README-Jenkins](./README-Jenkins.md) for requirements and instructions on how to run the 0-bootstrap step. Using Jenkins requires a few manual steps, including configuring connectivity with your current Jenkins Master environment.
-
-## 0-bootstrap usage to deploy Cloud Build
-
-1. Change into 0-bootstrap folder
-1. Copy tfvars by running `cp terraform.example.tfvars terraform.tfvars` and update `terraform.tfvars` with values from your environment.
-1. Run `terraform init`
-1. Run `terraform plan` and review output
-1. To run terraform-validator steps please follow the [instructions](https://github.com/forseti-security/policy-library/blob/master/docs/user_guide.md#install-terraform-validator) in the **Install Terraform Validator** section and install version `2021-03-22`. You will also need to rename the binary from `terraform-validator-<your-platform>` to `terraform-validator`.
```
Comment (daniel-cit) [inline 0-bootstrap/README.md:41]: Can we keep the  instructions on how to run terraform-validator with the bootstrap module?

File: 0-bootstrap/README.md [hunk 4]
```diff
@@ -1,59 +1,146 @@
...
+href="https://github.com/terraform-google-modules/terraform-example-foundation/tree/master/4-projects">4-projects</a></td>
+<td>Set up a folder structure and projects for applications, which are
+connected as service projects to the shared VPC created in the previous
+stage.</td>
+</tr>
+</tbody>
+</table>
+
+For an overview of the architecture and the parts, see the
+`[terraform-example-foundation` README file](https://github.com/terraform-google-modules/terraform-example-foundation).
```
Comment (daniel-cit) [inline 0-bootstrap/README.md:51]:
```suggestion
`terraform-example-foundation` [README](https://github.com/terraform-google-modules/terraform-example-foundation) file.
```

**Comment by daniel-cit** [comment]
Lint check failed with

```
Step #1 - "lint": Checking for trailing whitespace
Step #1 - "lint": ./0-bootstrap/README.md:85:   -  The `roles/resourcemanager.folderCreator` role.  
Step #1 - "lint": ./0-bootstrap/README.md:88:represented by the `org_project_creators` variable.  
Step #1 - "lint": ./0-bootstrap/README.md:118:   `cp backend.tf.example backend.tf` 
Step #1 - "lint": ./0-bootstrap/README.md:140:'s/UPDATE_ME/GCS_BUCKET_NAME/' $i; done` 
Step #1 - "lint": ./0-bootstrap/README.md:183:during a Terraform `apply` command:  
Step #1 - "lint": ./0-bootstrap/README.md:185:your allotted project quota`.   
Step #1 - "lint": Error: Trailing whitespace found in the lines above.
```

File: 0-bootstrap/README.md [hunk 5]
```diff
@@ -1,59 +1,152 @@
...
+</tbody>
+</table>
+
+For an overview of the architecture and the parts, see the
+[terraform-example-foundation README](https://github.com/terraform-google-modules/terraform-example-foundation)
+file.
+
+## Purpose
+
+The purpose of this step is to [...].
```
Comment (bharathkkb) [inline 0-bootstrap/README.md:53]:
```suggestion
The purpose of this step is to bootstrap a GCP organization, creating all the required resources & permissions to start using the Cloud Foundation Toolkit (CFT). This step also configures a CICD pipeline for foundations code in subsequent stages. The CICD pipeline can use either Cloud Build & Cloud Source Repos or Jenkins & your own Git repos (which might live on-prem).
```

File: 0-bootstrap/README.md [hunk 6]
```diff
@@ -1,59 +1,152 @@
...
+[Google Cloud security foundations guide](https://services.google.com/fh/files/misc/google-cloud-security-foundations-guide.pdf)
+(PDF). The following table lists the parts of the guide.
+
+<table>
+<tbody>
+<tr>
+<td>0-bootstrap (this file)</td>
+<td>Bootstraps a Google Cloud organization, creating all the required resources
+and permissions to start using the Cloud Foundation Toolkit (CFT). This
+step also configures a CI/CD pipeline for foundations code in subsequent
```
Comment (daniel-cit) [inline 0-bootstrap/README.md:14]: At this point we used `CI/CD` and in the other parts of this document we use `CICD` .
Which one should be the one to be the used?

File: 0-bootstrap/README.md [hunk 7]
```diff
@@ -1,59 +1,152 @@
...
+Storage bucket to the `backend.tf` files. You can update all of these files with
+the following steps:
+
+1. Go to the `terraform-example-foundation` directory.
+1. Run the following command:
+
+   ```
+   for i in `find -name 'backend.tf'`; do sed -i
+   's/UPDATE_ME/GCS_BUCKET_NAME/' $i; done
+   ```
```
Comment (daniel-cit) [inline 0-bootstrap/README.md:146]: This command should be in a single line to work

```
for i in `find -name 'backend.tf'`; do sed -i 's/GCS_BUCKET_NAME/UPDATE_ME/' $i; done
```

**Review by daniel-cit — COMMENTED** [review]
The `sed` line needs to be fixed, otherwise LGTM

File: 0-bootstrap/README.md (hunk 6 above)
Comment (bharathkkb) [reply 0-bootstrap/README.md:14]: I think we should converge on using CI/CD but both terms are correct https://en.wikipedia.org/wiki/CI/CD

File: 0-bootstrap/README.md (hunk 7 above)
Comment (bharathkkb) [reply 0-bootstrap/README.md:145]: good catch, done

File: 0-bootstrap/README.md [hunk 8]
```diff
@@ -1,59 +1,151 @@
...
+backends as a part of the build process when the build is executed by Cloud
+Build. If you want to execute Terraform locally, you need to add your Cloud
+Storage bucket to the `backend.tf` files. You can update all of these files with
+the following steps:
+
+1. Go to the `terraform-example-foundation` directory.
+1. Run the following command:
+
+   ```
+   for i in `find -name 'backend.tf'`; do sed -i 's/GCS_BUCKET_NAME/UPDATE_ME/' $i; done
```
Comment (daniel-cit) [inline 0-bootstrap/README.md:144]:
```suggestion
   for i in `find -name 'backend.tf'`; do sed -i 's/UPDATE_ME/GCS_BUCKET_NAME/' $i; done
```

@bharathkkb My bad, I was testing the command and pasted the one I used to revert the change in the suggestion, sorry.

**Review by daniel-cit — APPROVED** [review]
LGTM

**Comment by mark1000** [comment]
I believe I've fixed the sed command, and I think I resolved the merge conflicts correctly. However, could someone re-review to make sure I have it right?
